components/compass-runtime-agent/internal/kyma/apiresources/rafter/clusterassetgrouprepository_test.go

```diff
@@ -42,39 +42,39 @@ func TestCreateDocsTopic(t *testing.T) {
 	t.Run("Should fail if k8s client returned error on Create", func(t *testing.T) {
 		// given
 		resourceInterfaceMock := &mocks.ResourceInterface{}
-		repository := NewDocsTopicRepository(resourceInterfaceMock)
+		repository := NewAssetGroupRepository(resourceInterfaceMock)
 
 		resourceInterfaceMock.On("Create", mock.Anything, metav1.CreateOptions{}).
 			Return(&unstructured.Unstructured{}, errors.New("some error"))
 
 		// when
-		err := repository.Create(createTestDocsTopicEntry())
+		err := repository.Create(createTestClusterAssetGroupEntry())
 
 		// then
 		require.Error(t, err)
 		resourceInterfaceMock.AssertExpectations(t)
 	})
 }
 
-func TestUpdateDocsTopic(t *testing.T) {
-	t.Run("Should update DocsTopic", func(t *testing.T) {
+func TestUpdateClusterAssetGroup(t *testing.T) {
+	t.Run("Should update ClusterAssetGroup", func(t *testing.T) {
 		// given
 		resourceInterfaceMock := &mocks.ResourceInterface{}
-		repository := NewDocsTopicRepository(resourceInterfaceMock)
+		repository := NewAssetGroupRepository(resourceInterfaceMock)
 
-		dt := createK8sDocsTopic()
+		dt := createK8sClusterAssetGroup()
```

Comment:
It's cool to sometimes find some `re`s for the old Remote Environments. It feels like finding easter eggs ^^ Anyway, I think it should be renamed to `ag` for the Asset Group instead of the Docs Topis.
There are four of them. I don't want to spam you with similar comments so just find the `dt :=` here